internal/disgo_impl.go

```diff
@@ -75,6 +75,11 @@ type DisgoImpl struct {
 	largeThreshold           int
 }
 
+// GetAllStickerPacks returns all the api.StickerPack(s) available to nitro subscribers
+func (d *DisgoImpl) GetAllStickerPacks() ([]*api.StickerPack, restclient.RestError) {
+	return make([]*api.StickerPack, 0), nil
```

Comment:
Shouldn't this (and below) be returning API request data?

Reply:
the routes aren't impld into the rest lib, and since topi is working on merging the rest lib into here I'm gonna wait until then to finish the rest side of things

Reply:
Ah, might be worth leaving this as a draft for now then

Reply:
ya, this is still a draft, but i just wanted opinions on the impl itself since its going to get copied into the refactored codebase